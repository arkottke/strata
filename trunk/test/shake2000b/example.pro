    1   17   17   Example - 150-ft layer; EQ: Diam  diam.acc               
    1    2    1      5.00     3855.     0.007     0.125     996.1     1.000 1.00
    2    2    1      5.00     3023.     0.014     0.125     882.0     1.000 1.00
    3    2    1     10.00     2807.     0.023     0.125     849.9     1.000 1.00
    4    2    1     10.00     2989.     0.028     0.125     877.0     1.000 1.00
    5    1    1     10.00     3625.     0.030     0.125     965.9     1.000 1.00
    6    1    1     10.00     3544.     0.035     0.125     955.0     1.000 1.00
    7    1    1     10.00     4300.     0.034     0.125    1052.0     1.000 1.00
    8    1    1     10.00     4244.     0.037     0.125    1045.1     1.000 1.00
    9    2    1     10.00     5409.     0.034     0.130    1157.0     1.000 1.00
   10    2    1     10.00     5272.     0.037     0.130    1142.2     1.000 1.00
   11    2    1     10.00     6296.     0.034     0.130    1248.2     1.000 1.00
   12    2    1     10.00     6211.     0.035     0.130    1239.7     1.000 1.00
   13    2    1     10.00     7366.     0.032     0.130    1350.1     1.000 1.00
   14    2    1     10.00     7300.     0.033     0.130    1344.0     1.000 1.00
   15    2    1     10.00     8581.     0.030     0.130    1457.2     1.000 1.00
   16    2    1     10.00    11304.     0.026     0.130    1672.5     1.000 1.00
   17         1              69630.     0.010     0.140    4000.0
