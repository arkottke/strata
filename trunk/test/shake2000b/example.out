  ***********************************************************
  * SHAKE  --   A COMPUTER PROGRAM FOR EARTHQUAKE RESPONSE  *
  *       ANALYSIS OF HORIZONTALLY LAYERED SITES            *
  *             by: Per B. Schnabel & John Lysmer -- 1970   *
  * ------------------------------------------------------- *
  * SHAKE91   : General cleanup and finalization of input/  *
  *             output format ... etc                       *
  *             by: I. M. Idriss & J. I. Sun                *
  *             December 1991                               *
  * ------------------------------------------------------- *
  * SHAKE2000:  Improved in FFT stability, response spectra *
  *             calculation stability, data format errors,  *
  *             consistent stress calculation (Hysterisis   *
  *             loops), SI and English units, etc.          *
  *             by N. Deng & F. Ostadan                     *
  *             January  2000 -- Release 1.0                *
  *             December 2004 -- Convert to BSCP by ND      *
  *             August   2006 -- Release 1.1 by ND          *
  ***********************************************************

SHAKE2000  Release 1.1.  Example Problem                                        

 This run is in English Units (kip-ft-sec)

  MAX. NUMBER OF TERMS IN FOURIER TRANSFORM =       65536
  NECESSARY LENGTH OF BLANK COMMON X        =      524306

******   OPTION  1  ***  READ RELATION BETWEEN SOIL PROPERTIES AND STRAIN 

 **********************
  MATERIAL TYPE NO. 1
 **********************

 CURVE NO.  1:      #1 modulus for clay (seed & sun 1989) upper range      
 CURVE NO.  2:        damping for clay (Idriss 1990) -                     


      CURVE NO. 1              CURVE NO. 2    
  ===================       ==================
    STRAIN    G/Gmax         STRAIN    DAMPING
  --------   -------        --------  --------
    0.0001     1.000         0.0001      0.24
    0.0003     1.000         0.0003      0.42
    0.0010     1.000         0.0010      0.80
    0.0030     0.981         0.0030      1.40
    0.0100     0.941         0.0100      2.80
    0.0300     0.847         0.0300      5.10
    0.1000     0.656         0.1000      9.80
    0.3000     0.438         0.3000     15.50
    1.0000     0.238         1.0000     21.00
    3.0000     0.144         3.1600     25.00
   10.0000     0.110        10.0000     28.00

 **********************
  MATERIAL TYPE NO. 2
 **********************

 CURVE NO.  3:      #2 modulus for sand (seed & idriss 1970) - upper Range 
 CURVE NO.  4:        damping for sand (Idriss 1990) - (about LRng from SI 


      CURVE NO. 3              CURVE NO. 4    
  ===================       ==================
    STRAIN    G/Gmax         STRAIN    DAMPING
  --------   -------        --------  --------
    0.0001     1.000         0.0001      0.24
    0.0003     1.000         0.0003      0.42
    0.0010     0.990         0.0010      0.80
    0.0030     0.960         0.0030      1.40
    0.0100     0.850         0.0100      2.80
    0.0300     0.640         0.0300      5.10
    0.1000     0.370         0.1000      9.80
    0.3000     0.180         0.3000     15.50
    1.0000     0.080         1.0000     21.00
    3.0000     0.050         3.0000     25.00
   10.0000     0.035        10.0000     28.00

 **********************
  MATERIAL TYPE NO. 3
 **********************

 CURVE NO.  5:       #3 ATTENUATION OF  ROCK  AVERAGE                      
 CURVE NO.  6:       DAMPING IN ROCK                                       


      CURVE NO. 5              CURVE NO. 6    
  ===================       ==================
    STRAIN    G/Gmax         STRAIN    DAMPING
  --------   -------        --------  --------
    0.0001     1.000         0.0001      0.40
    0.0003     1.000         0.0010      0.80
    0.0010     0.988         0.0100      1.50
    0.0030     0.953         0.1000      3.00
    0.0100     0.900         1.0000      4.60
    0.0300     0.810         0.0000      0.00
    0.1000     0.725         0.0000      0.00
    1.0000     0.550         0.0000      0.00

******   OPTION  2  ***  READ SOIL PROFILE   

 NEW SOIL PROFILE NO.   1      IDENTIFICATION     Example - 150-ft layer; EQ: Diam 
 NUMBER OF LAYERS                   17          DEPTH TO BEDROCK        150.00

  NO. TYPE  THICKNESS   DEPTH  EFF.PRESS.  MODULUS  DAMPING   UNIT WT.  SHEAR VEL
              (ft)      (ft)     (ksf)      (ksf)              (kcf)      (fps)
   1    2      5.00      2.50      0.31       3886.   0.050    0.125     1000.0
   2    2      5.00      7.50      0.94       3147.   0.050    0.125      900.0
   3    2     10.00     15.00      1.88       3147.   0.050    0.125      900.0
   4    2     10.00     25.00      3.12       3507.   0.050    0.125      950.0
   5    1     10.00     35.00      4.38       3886.   0.050    0.125     1000.0
   6    1     10.00     45.00      5.62       3886.   0.050    0.125     1000.0
   7    1     10.00     55.00      6.88       4702.   0.050    0.125     1100.0
   8    1     10.00     65.00      8.12       4702.   0.050    0.125     1100.0
   9    2     10.00     75.00      9.40       6829.   0.050    0.130     1300.0
  10    2     10.00     85.00     10.70       6829.   0.050    0.130     1300.0
  11    2     10.00     95.00     12.00       7920.   0.050    0.130     1400.0
  12    2     10.00    105.00     13.30       7920.   0.050    0.130     1400.0
  13    2     10.00    115.00     14.60       9092.   0.050    0.130     1500.0
  14    2     10.00    125.00     15.90       9092.   0.050    0.130     1500.0
  15    2     10.00    135.00     17.20      10345.   0.050    0.130     1600.0
  16    2     10.00    145.00     18.50      13093.   0.050    0.130     1800.0
  17   BASE                                  69630.   0.010    0.140     4000.0

  PERIOD = 0.50 FROM AVERAGE SHEAR VELOCITY =   1198.

 MAXIMUM AMPLIFICATION =  13.82
 FOR FREQUENCY         =   2.32 C/SEC.
     PERIOD            =   0.43 SEC.

******   OPTION  3  ***  READ INPUT MOTION   

  FILE NAME FOR INPUT MOTION =  diam.acc                     
    NO. OF INPUT ACC. POINTS =  2000
   NO. OF POINTS USED IN FFT =  4096
        NO. OF HEADING LINES =     3
      NO. OF POINTS PER LINE =     8
  TIME STEP FOR INPUT MOTION = 0.0200
  FORMAT FOR OF TIME HISTORY = (8f10.6)    


 ***** H E A D E R   
"Loma P. Eqk","Diamond Hts","H1_90","init. vel:"," .307 c/s","disp: -0.016 cm"  
"Total No. of Points :",2000,"@ DT =",.02                                       
"Peak Acceleration (g) =",.1128945,"@ Time (sec) :",10.92                       
 **  FIRST & LAST 5 LINES OF INPUT MOTION *****

     1     -0.001694   -0.001668   -0.000086   -0.001356   -0.000678    0.000700   -0.001209   -0.000604
     2      0.000730    0.000737    0.002496    0.004583    0.001644    0.001377    0.002408   -0.000352
     3     -0.001073   -0.000359   -0.000486    0.000344    0.000767   -0.002507   -0.003164   -0.002890
     4     -0.004086    0.000143    0.004340    0.003943    0.002350   -0.001087   -0.002345    0.001716
     5     -0.001943   -0.007436   -0.004493    0.000827    0.002915    0.003241    0.003055    0.002658
   ........ INPUT MOTION READ NOT ECHOED...........
   246      0.004209    0.003945    0.004607    0.004357    0.004119    0.003705    0.002579    0.001774
   247      0.001345    0.002029    0.003633    0.004110    0.004086    0.004056    0.004223    0.002753
   248      0.001352    0.001639    0.001365    0.001602    0.001387    0.001505    0.000402   -0.001137
   249     -0.001180   -0.001222   -0.001202   -0.001212   -0.001217   -0.001072    0.000252   -0.000560
   250     -0.000910    0.000950    0.001431    0.001505    0.001475    0.001494    0.001485    0.001486

 MAXIMUM ACCELERATION =    0.11289
 AT TIME              =   10.92 SEC
 THE VALUES WILL BE MULTIPLIED BY A FACTOR =   0.885779
 TO GIVE NEW MAXIMUM ACCELERATION          =    0.10000
 MEAN SQUARE FREQUENCY =        2.52 C/SEC.
  MAX ACCELERATION =    0.10000 FOR FREQUENCIES REMOVED ABOVE      25.00 C/SEC.

******   OPTION  4  ***  READ WHERE OBJECT MOTION IS GIVEN 

  OBJECT MOTION IN LAYER NUMBER  17 OUTCROPPING

******   OPTION  5  ***  OBTAIN STRAIN COMPATIBLE SOIL PROPERTIES

      MAXIMUM NUMBER OF ITERATIONS                =    8
      FACTOR FOR UNIFORM STRAIN IN TIME DOMAIN    =  0.50
	STRESS CALCULATION CONTROL KEY              =    1

     EARTHQUAKE   -   diam.acc                     
     SOIL PROFILE -    Example - 150-ft layer; EQ: Diam 

  ITERATION NUMBER    1

  VALUES IN TIME DOMAIN

 NO TYPE DEPTH  UNIFRM.   <---- DAMPING ---->    <---- SHEAR MODULUS ----->   G/Go
         (FT)   STRAIN   NEW    USED    ERROR      NEW      USED     ERROR    RATIO
--- ---- -----  ------- -----  ------  -------   -------  --------  -------   -----
  1  2     2.5  0.00074  0.007  0.050   -607.5    3856.3    3885.6     -0.8   1.000
  2  2     7.5  0.00275  0.014  0.050   -269.6    3028.9    3147.3     -3.9   1.000
  3  2    15.0  0.00546  0.021  0.050   -138.5    2849.1    3147.3    -10.5   1.000
  4  2    25.0  0.00803  0.025  0.050    -96.5    3051.0    3506.8    -14.9   1.000
  5  1    35.0  0.00990  0.028  0.050    -79.3    3657.6    3885.6     -6.2   1.000
  6  1    45.0  0.01235  0.032  0.050    -54.2    3586.2    3885.6     -8.3   1.000
  7  1    55.0  0.01203  0.032  0.050    -56.9    4349.8    4701.6     -8.1   1.000
  8  1    65.0  0.01366  0.035  0.050    -44.8    4298.8    4701.6     -9.4   1.000
  9  2    75.0  0.01040  0.029  0.050    -73.5    5753.7    6829.3    -18.7   1.000
 10  2    85.0  0.01129  0.031  0.050    -63.7    5646.3    6829.3    -21.0   1.000
 11  2    95.0  0.01039  0.029  0.050    -73.6    6674.4    7920.4    -18.7   1.000
 12  2   105.0  0.01093  0.030  0.050    -67.5    6598.1    7920.4    -20.0   1.000
 13  2   115.0  0.00988  0.028  0.050    -79.5    7738.8    9092.3    -17.5   1.000
 14  2   125.0  0.01013  0.028  0.050    -76.9    7706.7    9092.3    -18.0   1.000
 15  2   135.0  0.00902  0.027  0.050    -86.5    8890.4   10345.0    -16.4   1.000
 16  2   145.0  0.00716  0.024  0.050   -107.4   11529.0   13092.9    -13.6   1.000

  ITERATION NUMBER    2

  ITERATION NUMBER    3

  ITERATION NUMBER    4

  ITERATION NUMBER    5

  ITERATION NUMBER    6

  ITERATION NUMBER    7

  ITERATION NUMBER    8

  VALUES IN TIME DOMAIN

 NO TYPE DEPTH  UNIFRM.   <---- DAMPING ---->    <---- SHEAR MODULUS ----->   G/Go
         (FT)   STRAIN   NEW    USED    ERROR      NEW      USED     ERROR    RATIO
--- ---- -----  ------- -----  ------  -------   -------  --------  -------   -----
  1  2     2.5  0.00077  0.007  0.007      0.0    3855.2    3855.2      0.0   0.992
  2  2     7.5  0.00295  0.014  0.014      0.0    3022.9    3022.9      0.0   0.960
  3  2    15.0  0.00633  0.023  0.023      0.0    2806.7    2806.7      0.0   0.892
  4  2    25.0  0.00975  0.028  0.028      0.0    2988.9    2988.8      0.0   0.852
  5  1    35.0  0.01098  0.030  0.030      0.0    3625.3    3625.3      0.0   0.933
  6  1    45.0  0.01402  0.035  0.035      0.0    3544.1    3544.1      0.0   0.912
  7  1    55.0  0.01360  0.034  0.034      0.0    4300.4    4300.4      0.0   0.915
  8  1    65.0  0.01565  0.037  0.037      0.0    4244.1    4244.1      0.0   0.903
  9  2    75.0  0.01354  0.034  0.034      0.0    5409.2    5409.2      0.0   0.792
 10  2    85.0  0.01504  0.037  0.037      0.0    5272.4    5272.4      0.0   0.772
 11  2    95.0  0.01334  0.034  0.034      0.0    6295.7    6295.8      0.0   0.795
 12  2   105.0  0.01411  0.035  0.035      0.0    6210.8    6210.9      0.0   0.784
 13  2   115.0  0.01232  0.032  0.032      0.0    7366.1    7366.1      0.0   0.810
 14  2   125.0  0.01280  0.033  0.033      0.0    7299.6    7299.6      0.0   0.803
 15  2   135.0  0.01114  0.030  0.030      0.0    8580.7    8580.6      0.0   0.829
 16  2   145.0  0.00864  0.026  0.026      0.0   11304.4   11304.4      0.0   0.863

  VALUES IN TIME DOMAIN

  LAYER  TYPE      THICKNESS         DEPTH      MAX STRAIN          TIME        MAX STRESS        TIME
                       FT             FT         PERCENT            SEC            PSF            SEC

     1     2          5.00            2.50       0.00154           11.28          59.50          11.28
     2     2          5.00            7.50       0.00590           11.30         178.27          11.28
     3     2         10.00           15.00       0.01266           11.30         354.64          11.28
     4     2         10.00           25.00       0.01950           11.30         582.72          11.28
     5     1         10.00           35.00       0.02196           11.30         797.98          11.28
     6     1         10.00           45.00       0.02804           11.30         997.10          11.28
     7     1         10.00           55.00       0.02721           11.30        1176.06          11.28
     8     1         10.00           65.00       0.03129           11.30        1334.06          11.28
     9     2         10.00           75.00       0.02708           11.30        1471.65          11.28
    10     2         10.00           85.00       0.03008           11.30        1589.53          11.28
    11     2         10.00           95.00       0.02669           11.30        1683.28          11.28
    12     2         10.00          105.00       0.02822           11.30        1753.80          11.28
    13     2         10.00          115.00       0.02464           11.52        1820.45          11.52
    14     2         10.00          125.00       0.02560           11.52        1876.67          11.52
    15     2         10.00          135.00       0.02227           11.52        1920.02          11.52
    16     2         10.00          145.00       0.01727           11.54        1952.39          11.52

  PERIOD = 0.54 FROM AVERAGE SHEAR VELOCITY =   1112.

 MAXIMUM AMPLIFICATION =  20.52
 FOR FREQUENCY         =   2.11 C/SEC.
     PERIOD            =   0.47 SEC.

******   OPTION  6  ***  COMPUTE MOTION IN NEW SUBLAYERS         

   EARTHQUAKE - diam.acc                     
  SOIL DEPOSIT -    Example - 150-ft layer; EQ: Diam 
   LAYER       LAYER       DEPTH     MAX. ACC.      TIME     MEAN SQ. FR.    ACC. RATIO     TH SAVED
    NO.        TYPE         FT          G           SEC        C/SEC         QUIET ZONE    ACC. RECORD

     1        OUTCR.         0.0     0.19043       11.28        2.42            0.000           512
     2        WITHIN         5.0     0.19008       11.28        2.40            0.000             0
     3        WITHIN        10.0     0.18876       11.28        2.35            0.000             0
     4        WITHIN        20.0     0.18261       11.28        2.23            0.000             0
     5        WITHIN        30.0     0.17211       11.28        2.19            0.000             0
     6        WITHIN        40.0     0.15948       11.28        2.19            0.000             0
     7        WITHIN        50.0     0.14289       11.28        2.17            0.000             0
     8        WITHIN        60.0     0.12653       11.28        2.13            0.000             0
     9        WITHIN        70.0     0.11051       11.52        2.12            0.000             0
    10        WITHIN        80.0     0.09840       11.54        2.14            0.000             0
    11        WITHIN        90.0     0.08997       11.56        2.19            0.000             0
    12        WITHIN       100.0     0.08268       11.56        2.24            0.000             0
    13        WITHIN       110.0     0.08558       10.94        2.31            0.000             0
    14        WITHIN       120.0     0.08545       10.94        2.39            0.000             0
    15        WITHIN       130.0     0.08200       10.94        2.45            0.000             0

******   OPTION  6  ***  COMPUTE MOTION IN NEW SUBLAYERS         

   EARTHQUAKE - diam.acc                     
  SOIL DEPOSIT -    Example - 150-ft layer; EQ: Diam 
   LAYER       LAYER       DEPTH     MAX. ACC.      TIME     MEAN SQ. FR.    ACC. RATIO     TH SAVED
    NO.        TYPE         FT          G           SEC        C/SEC         QUIET ZONE    ACC. RECORD

    16        WITHIN       140.0     0.07769       10.92        2.48            0.000             0
    17        WITHIN       150.0     0.07616       10.92        2.48            0.000           512
    17        OUTCR.       150.0     0.10000       10.92        2.52            0.000           512

******   OPTION  7  ***  COMPUTE STRESS/STRAIN HISTORY             

  COMPUTE STRESS OR STRAIN HISTORY AT THE TOP OF  LAYER    4
  IDENTIFICATION - stress in level 4             

  COMPUTE STRESS OR STRAIN HISTORY AT THE TOP OF  LAYER    4
  IDENTIFICATION - strain in level 4             

******   OPTION  7  ***  COMPUTE STRESS/STRAIN HISTORY             

  COMPUTE STRESS OR STRAIN HISTORY AT THE TOP OF  LAYER    8
  IDENTIFICATION - stress in level 8             

  COMPUTE STRESS OR STRAIN HISTORY AT THE TOP OF  LAYER    8
  IDENTIFICATION - strain in level 8             

******   OPTION  9  ***  COMPUTE RESPONSE SPECTRUM               

  RESPONSE SPECTRUM ANALYSIS FOR LAYER NUMBER       1  OUTCROPPING
  RESPONSES ARE CALCULATED FOR DAMPING 
     0.050
  RESPONSES ARE CALCULATED USING CONSTANT TIME STEP

 Response Spectra Computation: 
  The Shortest Period =                  0.01 (Second)
  The Longest Period =                  10.00 (Second)
  Total Number of Calculated Points:   140
  All Points Are Evenly Distributed in Log-Scale


     DAMPING RATIO =   0.050


      SPECTRAL VALUES --
      [Acceleration of gravity used =   32.17]
 diam.acc                        Example - 150-ft layer; EQ:  DAMPING RATIO = 0.05
     NO.    PERIOD     REL. DISP.      REL. VEL.   PSU.REL.VEL.      ABS. ACC.   PSU.ABS.ACC.     FREQ.
       1     0.010        0.00002        0.00022        0.00974        0.19040        0.19022   100.000
       2     0.011        0.00002        0.00029        0.01019        0.18949        0.18936    95.152
       3     0.011        0.00002        0.00038        0.01074        0.18995        0.18988    90.539
       4     0.012        0.00002        0.00043        0.01128        0.18991        0.18984    86.149
       5     0.012        0.00002        0.00044        0.01187        0.19012        0.19007    81.973
       6     0.013        0.00003        0.00048        0.01249        0.19030        0.19025    77.999
       7     0.013        0.00003        0.00053        0.01314        0.19046        0.19040    74.217
       8     0.014        0.00003        0.00059        0.01382        0.19063        0.19056    70.619
       9     0.015        0.00003        0.00066        0.01454        0.19091        0.19084    67.195
      10     0.016        0.00004        0.00080        0.01531        0.19128        0.19118    63.937
      11     0.016        0.00004        0.00091        0.01606        0.19088        0.19081    60.838
      12     0.017        0.00005        0.00096        0.01697        0.19191        0.19189    57.888
      13     0.018        0.00005        0.00086        0.01796        0.19346        0.19319    55.082
      14     0.019        0.00006        0.00086        0.01869        0.19173        0.19132    52.411
      15     0.020        0.00006        0.00072        0.01950        0.19041        0.18996    49.870
      16     0.021        0.00007        0.00112        0.02029        0.18857        0.18801    47.452
      17     0.022        0.00007        0.00131        0.02107        0.18608        0.18577    45.152
      18     0.023        0.00008        0.00167        0.02242        0.18787        0.18813    42.963
      19     0.024        0.00009        0.00245        0.02399        0.19156        0.19155    40.880
      20     0.026        0.00010        0.00260        0.02503        0.19014        0.19017    38.898
      21     0.027        0.00011        0.00274        0.02639        0.19089        0.19078    37.012
      22     0.028        0.00012        0.00255        0.02765        0.19037        0.19017    35.218
      23     0.030        0.00014        0.00254        0.02904        0.19030        0.19009    33.510
      24     0.031        0.00015        0.00293        0.03051        0.19028        0.19001    31.886
      25     0.033        0.00017        0.00330        0.03206        0.19009        0.18995    30.340
      26     0.035        0.00019        0.00361        0.03365        0.18990        0.18975    28.869
      27     0.036        0.00021        0.00393        0.03540        0.19002        0.18990    27.469
      28     0.038        0.00023        0.00422        0.03727        0.19043        0.19029    26.138
      29     0.040        0.00025        0.00467        0.03917        0.19043        0.19027    24.870
      30     0.042        0.00028        0.00517        0.04116        0.19041        0.19026    23.665
      31     0.044        0.00031        0.00588        0.04340        0.19102        0.19088    22.517
      32     0.047        0.00034        0.00658        0.04566        0.19127        0.19107    21.426
      33     0.049        0.00037        0.00730        0.04792        0.19105        0.19082    20.387
      34     0.052        0.00041        0.00817        0.05049        0.19102        0.19128    19.399
      35     0.054        0.00046        0.00885        0.05336        0.19193        0.19236    18.458
      36     0.057        0.00051        0.00941        0.05631        0.19291        0.19315    17.563
      37     0.060        0.00057        0.01000        0.05986        0.19555        0.19538    16.712
      38     0.063        0.00062        0.01200        0.06235        0.19422        0.19366    15.902
      39     0.066        0.00069        0.01479        0.06563        0.19388        0.19395    15.131
      40     0.069        0.00076        0.01659        0.06908        0.19394        0.19426    14.397
      41     0.073        0.00086        0.02138        0.07426        0.19843        0.19868    13.699
      42     0.077        0.00100        0.02675        0.08220        0.20918        0.20926    13.035
      43     0.081        0.00111        0.03200        0.08626        0.21068        0.20897    12.403
      44     0.085        0.00121        0.03912        0.08942        0.20593        0.20612    11.802
      45     0.089        0.00138        0.04520        0.09727        0.21535        0.21334    11.229
      46     0.094        0.00163        0.05391        0.10953        0.22887        0.22857    10.685
      47     0.098        0.00179        0.07051        0.11414        0.22987        0.22665    10.167
      48     0.103        0.00214        0.07150        0.12986        0.24563        0.24537     9.674
      49     0.109        0.00251        0.09302        0.14545        0.26310        0.26150     9.205
      50     0.114        0.00290        0.08613        0.15953        0.27651        0.27290     8.759
      51     0.120        0.00324        0.09247        0.16984        0.27343        0.27646     8.334
      52     0.126        0.00347        0.08035        0.17299        0.26938        0.26793     7.930
      53     0.133        0.00377        0.08559        0.17872        0.26608        0.26340     7.546
      54     0.139        0.00428        0.09190        0.19295        0.26774        0.27057     7.180
      55     0.146        0.00487        0.10388        0.20915        0.27939        0.27907     6.832
      56     0.154        0.00503        0.10879        0.20564        0.26329        0.26108     6.501
      57     0.162        0.00543        0.12018        0.21099        0.25735        0.25489     6.185
      58     0.170        0.00644        0.12045        0.23817        0.27154        0.27378     5.886
      59     0.179        0.00680        0.13342        0.23930        0.26352        0.26174     5.600
      60     0.188        0.00757        0.14549        0.25348        0.26484        0.26381     5.329
      61     0.197        0.00866        0.17024        0.27578        0.27497        0.27310     5.070
      62     0.207        0.00938        0.17245        0.28442        0.26826        0.26801     4.825
      63     0.218        0.01084        0.22912        0.31274        0.28014        0.28041     4.591
      64     0.229        0.01273        0.28640        0.34944        0.29674        0.29812     4.368
      65     0.241        0.01540        0.29133        0.40219        0.32517        0.32649     4.156
      66     0.253        0.02070        0.36078        0.51437        0.39793        0.39730     3.955
      67     0.266        0.02554        0.40662        0.60385        0.44414        0.44381     3.763
      68     0.279        0.02802        0.48530        0.63034        0.43970        0.44082     3.581
      69     0.294        0.03252        0.50416        0.69610        0.46078        0.46321     3.407
      70     0.308        0.03494        0.51235        0.71175        0.45366        0.45066     3.242
      71     0.324        0.04318        0.62849        0.83695        0.50601        0.50424     3.085
      72     0.341        0.05759        0.84944        1.06211        0.60726        0.60887     2.935
      73     0.358        0.07198        1.07665        1.26306        0.69604        0.68896     2.793
      74     0.376        0.09105        1.35607        1.52019        0.79635        0.78902     2.657
      75     0.395        0.10364        1.55169        1.64662        0.81550        0.81320     2.529
      76     0.416        0.11859        1.75455        1.79283        0.84405        0.84249     2.406
      77     0.437        0.12291        1.79119        1.76795        0.79607        0.79052     2.289
      78     0.459        0.13149        1.88369        1.79968        0.77516        0.76569     2.178
      79     0.482        0.13078        1.78650        1.70325        0.69677        0.68953     2.073
      80     0.507        0.12181        1.58934        1.50943        0.58280        0.58144     1.972
      81     0.533        0.13891        1.69145        1.63796        0.60447        0.60036     1.877
      82     0.560        0.14931        1.81021        1.67515        0.58578        0.58423     1.786
      83     0.589        0.13623        1.62929        1.45434        0.48562        0.48263     1.699
      84     0.619        0.13686        1.56181        1.39020        0.44407        0.43897     1.617
      85     0.650        0.16289        1.66607        1.57441        0.47638        0.47304     1.538
      86     0.683        0.17175        1.77425        1.57955        0.45406        0.45157     1.464
      87     0.718        0.16115        1.61141        1.41025        0.38812        0.38363     1.393
      88     0.755        0.13895        1.43323        1.15700        0.30338        0.29948     1.325
      89     0.793        0.12172        1.28470        0.96443        0.23939        0.23753     1.261
      90     0.833        0.12079        1.26370        0.91066        0.21459        0.21341     1.200
      91     0.876        0.12447        1.30314        0.89289        0.20141        0.19910     1.142
      92     0.921        0.14142        1.31935        0.96532        0.20622        0.20482     1.086
      93     0.967        0.15025        1.25401        0.97585        0.19916        0.19701     1.034
      94     1.017        0.15421        1.15156        0.95299        0.18450        0.18307     0.984
      95     1.069        0.15652        1.06218        0.92041        0.17023        0.16824     0.936
      96     1.123        0.16518        0.99534        0.92421        0.16244        0.16075     0.891
      97     1.180        0.17542        1.06092        0.93395        0.15638        0.15456     0.847
      98     1.240        0.17671        1.14660        0.89521        0.14262        0.14097     0.806
      99     1.303        0.16845        1.19262        0.81199        0.12322        0.12167     0.767
     100     1.370        0.15175        1.19320        0.69601        0.10054        0.09923     0.730
     101     1.440        0.15189        1.14975        0.66287        0.09076        0.08993     0.695
     102     1.513        0.15512        1.07961        0.64414        0.08402        0.08315     0.661
     103     1.590        0.15920        1.00810        0.62903        0.07822        0.07726     0.629
     104     1.671        0.16264        0.95698        0.61149        0.07262        0.07147     0.598
     105     1.756        0.16947        0.93391        0.60626        0.06834        0.06742     0.569
     106     1.846        0.18058        0.93122        0.61469        0.06614        0.06504     0.542
     107     1.940        0.19281        0.92883        0.62451        0.06377        0.06288     0.516
     108     2.039        0.20094        0.91606        0.61930        0.06024        0.05933     0.491
     109     2.143        0.20547        0.89817        0.60256        0.05577        0.05493     0.467
     110     2.252        0.21429        0.87568        0.59796        0.05216        0.05187     0.444
     111     2.366        0.23904        0.93451        0.63468        0.05287        0.05238     0.423
     112     2.487        0.25157        0.97242        0.63555        0.05061        0.04991     0.402
     113     2.614        0.25061        0.96459        0.60244        0.04585        0.04502     0.383
     114     2.747        0.24486        0.91990        0.56007        0.04077        0.03982     0.364
     115     2.887        0.24185        0.85748        0.52638        0.03580        0.03561     0.346
     116     3.034        0.23653        0.79885        0.48983        0.03177        0.03153     0.330
     117     3.189        0.21902        0.75125        0.43158        0.02670        0.02644     0.314
     118     3.351        0.23068        0.70868        0.43252        0.02543        0.02521     0.298
     119     3.522        0.23120        0.66291        0.41248        0.02309        0.02288     0.284
     120     3.701        0.21593        0.62083        0.36656        0.02021        0.01934     0.270
     121     3.890        0.21378        0.61205        0.34531        0.01823        0.01734     0.257
     122     4.088        0.20476        0.59448        0.31471        0.01597        0.01504     0.245
     123     4.296        0.19280        0.57120        0.28197        0.01373        0.01282     0.233
     124     4.515        0.18318        0.54726        0.25491        0.01183        0.01103     0.221
     125     4.745        0.18507        0.53075        0.24505        0.01040        0.01009     0.211
     126     4.987        0.19406        0.52746        0.24450        0.00964        0.00958     0.201
     127     5.241        0.22183        0.52740        0.26594        0.01004        0.00991     0.191
     128     5.508        0.24339        0.53221        0.27764        0.00989        0.00984     0.182
     129     5.789        0.25442        0.54147        0.27615        0.00943        0.00932     0.173
     130     6.084        0.26426        0.55343        0.27292        0.00881        0.00876     0.164
     131     6.394        0.27020        0.56592        0.26553        0.00816        0.00811     0.156
     132     6.720        0.26518        0.57699        0.24796        0.00725        0.00721     0.149
     133     7.062        0.26216        0.58530        0.23326        0.00648        0.00645     0.142
     134     7.422        0.25354        0.59026        0.21464        0.00569        0.00565     0.135
     135     7.800        0.23939        0.59187        0.19284        0.00486        0.00483     0.128
     136     8.197        0.22792        0.59061        0.17470        0.00419        0.00416     0.122
     137     8.615        0.21313        0.58719        0.15544        0.00357        0.00352     0.116
     138     9.054        0.19221        0.58241        0.13339        0.00290        0.00288     0.110
     139     9.515        0.17007        0.57698        0.11230        0.00237        0.00231     0.105
     140    10.000        0.16333        0.57149        0.10263        0.00210        0.00200     0.100

          VALUES IN PERIOD RANGE .1 TO 2.5 SEC.
               AREA OF ACC. RESPONSE SPECTRUM   =       0.510
               AREA OF VEL. RESPONSE SPECTRUM   =       2.560
               MAX. ACCELERATION RESPONSE VALUE =       0.844
               MAX. VELOCITY     RESPONSE VALUE =       1.884

******   OPTION 10  ***  COMPUTE AMPLIFICATION FUNCTION            

    AMPLIFICATION SPECTRUM BETWEEN LAYER  17 AND   1
 REFERENCE MOTION AT LAYER   17  WITHIN      
    OUTPUT MOTION AT LAYER    1  OUTCROPPING 

** AMPLIFICATION FUNCTION IS WRITTEN ON FILE example.tfs                                                 

 MAXIMUM AMPLIFICATION =  20.52
 FOR FREQUENCY         =   2.11 C/SEC.
     PERIOD            =   0.47 SEC.

******   OPTION 10  ***  COMPUTE AMPLIFICATION FUNCTION            

    AMPLIFICATION SPECTRUM BETWEEN LAYER  17 AND  17
 REFERENCE MOTION AT LAYER   17  WITHIN      
    OUTPUT MOTION AT LAYER   17  OUTCROPPING 

** AMPLIFICATION FUNCTION IS WRITTEN ON FILE example.tfs                                                 

 MAXIMUM AMPLIFICATION =   5.97
 FOR FREQUENCY         =   2.11 C/SEC.
     PERIOD            =   0.47 SEC.

******   OPTION 11  ***       FOURIER SPECTRUM OF COMPUTED MOTION
  LAYER NUMBER    1  OUTCROPPING 
  LAYER NUMBER   17  WITHIN      

** FOURIER SPECTRA ARE WRITTEN ON FILE example.fsm                                                 




 *** END OF INPUT FILE  *** 

 *** NORMAL TERMINATION	*** 

